### DISCUSSION TITLE
Received a GOAWAY with error code ENHANCE_YOUR_CALM and debug data equal to "too_many_pings"

### DISCUSSION THREAD

**Discussion by sloth2012** [milvus-io/milvus#5626] [Q&A and General discussion]
with python multiprocess，meet this question。

it can not keep recurring。 it works well for serveral times. I searched this error, and found it may be caused by grpc.

my grpc version is 1.37.0

**Comment by yhmo** [comment]
Use multiprocess in pymilvus may encounter problems, please refer this page: https://milvus.io/docs/v0.6.0/operational_faq.md#Why-does-my-multiprocessing-program-fail

Could you use multithreads instead?

**Reply by sloth2012** [reply]
I build client in the main process， and each child process create its own client，is that ok？

**Reply by yhmo** [reply]
> I build client in the main process， and each child process create its own client，is that ok？

Just try. If still encounter error, please refer this issue #4863

**Reply by sloth2012** [reply]
3ks, I found that it works well even this message appears。
